src/ert/cli/model_factory.py

```diff
@@ -43,16 +47,20 @@ def create_model(ert, ensemble_size, current_case_name, args, id_):
     return model
 
 
-def _setup_single_test_run(ert, id_):
-    simulations_argument = {"active_realizations": [True]}
+def _setup_single_test_run(ert, current_case_name, id_):
+    simulations_argument = {
+        "active_realizations": [True],
+        "current_case": current_case_name,
+    }
     model = SingleTestRun(simulations_argument, ert, id_)
```

Comment:
Nitpicky but we can just return here without creating the variable model. Same goes for several functions in this file.